modules/ROOT/pages/api-groups-add-sla-tiers.adoc

```diff
@@ -0,0 +1,30 @@
+= Add SLA Tiers To API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can add multiple SLA tiers to an API Group instance.
+
+To add an SLA tier:
+
+. From the left menu in API Manager, click *API Group*.
++
+The API Groups details page appears.
+. Expand the API Group list and click the API Group instance to which you want to add the SLA tier.
+. From the left menu, click *SLA Tiers* and then click *Add SLA Tiers* on the details page.
++
+. The illustration shows the Add SLA Tier window.
+image:api-groups-add-sla-tier.png[Add SLA Tier]
+. Click *Continue*.
++
+The Adding limits to SLA tier page appears.
+. At the top of the SLA tier details area, click `Default` if you want the SLA tier to apply to API instances in the API Group instance.
```

Comment:
we should specify that an `Individual` limit will override the default limit for the individual API

Reply:
We do. In the SLA tier main page:
If you have configured a default SLA limit at the API Group instance level and an individual SLA limit at the API instance level, the individual SLA takes precedence over the default limit.